src/intellisense/pythonKernelCompletionProvider.node.ts

```diff
@@ -30,6 +30,11 @@ import { IInteractiveWindowProvider } from '../interactive-window/types';
 import { Settings } from '../platform/common/constants';
 import { INotebookCompletion } from './types';
 
+let IntellisenseTimeout = Settings.IntellisenseTimeout;
+export function setIntellisenseTimeout(timeoutMs: number) {
```

Comment:
Cant use process.ENV anymore (except for node specific code). Tests were not node specific so used this instead.

Reply:
Curious why are we preventing the use of `process.env`?
If we allowed it, then we wouldn't need this change

Reply:
Not entirely sure, but if we want we can always inject env variables into the web bundle using the `Define` section in webpack. Again, not sure why we're disabling the use of `process.env`